app/models/notification.rb

```diff
@@ -8,12 +10,12 @@ class << self
     define_cursor_paginator :cursored_for, :for
     define_cursor_paginator :cursored_for_type, :for_type
 
-    def for(recipient, options={})
-      self.where(options.merge!(recipient: recipient)).order(:created_at).reverse_order
+    def for(recipient, options = {})
+      where(options.merge!(recipient: recipient)).order(:created_at).reverse_order
     end
 
-    def for_type(recipient, type, options={})
-      self.where(options.merge!(recipient: recipient)).where('LOWER(target_type) = ?', type).order(:created_at).reverse_order
+    def for_type(recipient, type, options = {})
```

Comment:
the modern Ruby thing to do would be:
```suggestion
    def for_type(recipient, type, **kwargs)
```

`kwargs` will be your usual Hash

Reply:
b1380922